### PR TITLE
replaced input with textarea, added a shift+enter functionality

### PR DESCRIPTION
**Pull request by tomoversluizen**

### CODE REVIEW

File: src/message-input/index.jsx [hunk 1]
```diff
@@ -79,6 +80,12 @@ class MessageInput extends Component {
     );
   }
 
+  handleKeyPress(e) {
+    if (e.nativeEvent.keyCode === 13) {
+      if (this.nativeEvent.shiftKey);
```
Comment (sjaakluthart) [inline src/message-input/index.jsx]: Why did you add this line?

**Comment by sjaakluthart** [comment]
screen shot 2017-05-02 at 08 41 00

Styling is also broken, make sure the textarea cannot be dragged like this.

**Comment by sjaakluthart** [comment]
screen shot 2017-05-02 at 08 44 45

Also has errors.

**Comment by larstadema** [comment]
Personally I would split both the input and multiLineInput into seperate components. Makes your index file alot cleaner, and you can tweak logic per type

File: src/message-input/index.jsx [hunk 2]
```diff
@@ -99,11 +108,46 @@ class MessageInput extends Component {
...
+      />
+    );
+
+    if (multiline) {
+      input = (
+        <textarea
+          style={getInputStyle(leftButton, multiline, inputStyle)}
+          value={value}
+          onChange={onChange}
+          format="multiline"
```
Comment (larstadema) [inline src/message-input/index.jsx]: This property does not exist, why did you use it?

**Comment by larstadema** [comment]
Errors when typing in the input:

screen shot 2017-05-11 at 15 06 54

File: src/message-input/index.jsx (hunk 2 above)
Comment (larstadema) [inline src/message-input/index.jsx]: https://developer.mozilla.org/nl/docs/Web/HTML/Element/textarea
